monkey/infection_monkey/master/automated_master.py

```diff
@@ -201,7 +204,7 @@ def _run_pba(self, pba: Tuple[str, Dict]):
             self._telemetry_messenger.send_telemetry(PostBreachTelem(pba_data))
 
     def _can_propagate(self) -> bool:
-        return True
+        return should_propagate(self._control_channel.get_config(), self._current_depth)
```

Comment:
Since `_can_propagate()` does nothing other than wrap `should_propagate()`, should we just remove this function? It's only used in one place, so changing it back is simple.

Reply:
My guess is we maybe planned to fill it in with more complicated logic? I don't remember.